ssv/spectest/tests/runner/duties/newduty/duplicate_duty_finished.go

```diff
@@ -0,0 +1,65 @@
+package newduty
+
+import (
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// DuplicateDutyFinished is a test that runs the following scenario:
+// - Runner is assigned a duty
+// - Runner finishes the duty
+// - Runner is assigned the same duty again
+// TODO - Does it make sense that the runner starts? Does it make sense that duties with precon phase don't return an error?
```

Comment:
@alonmuroch

should I open an issue? it just seems to me that it is odd that duties that can't enter consensus phase return an error so late

Reply:
same duty should throw and error and shouldn't start IMO

Reply:
https://github.com/bloxapp/ssv-spec/issues/282

See if you agree with the issue, it is more general than what I wrote in the comment.